### PR TITLE
prunes en.json

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -1185,52 +1193,146 @@
...
+        "head": "Delete <strong>{{label}}</strong>?",
+        "body": "Are you sure you wish to delete the sandbox <strong>{{label}}</strong>?"
+      }
+    }
+  },
+
+  "tenants": {
+    "heading": {
+      "SANDBOX": "Sandboxes",
+      "TENANT": "Tenants"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: any objection to changing the verbiage here?

image

Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: reads better to me

image

Comment (ftdholbrook) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: looks better to me

Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: 👍

**Pull request merged**